source/release_notes.rst

```diff
@@ -5,7 +5,77 @@ Release Notes
 
 .. contents:: :local:
 
-----
+
+
+--------
+
+ww201826
+========
+
+================================  =====
+Build ID                          celadon-userdebug 8.1.0 OPM4.171019.021.E1 eng.build.20180629.162304 test-keys
+Software Version                  Android version 8.1.0
+Mesa                              18.2.0-devel
+Kernel Flinger                    5.09
+Kernel Version                    4.14.50
+Download Link                     https://github.com/projectceladon/celadon-binary/tree/master/ww201826
+================================  =====
+
+Integrated Features
+-------------------
+In the WW24 release we have IVI also enabled , IVI is for Enabling CAR UI
```

Comment:
This is WW26 release, how about we put "The Car UI for IVI usage has been enabled since the WW release."?